frontend/src/components/Bill/Bill.jsx

```diff
@@ -4,7 +4,21 @@ import { useParams } from 'react-router-dom';
 import LocForm from "./LocForm";
 import './Bill.css';
 
-function getSponsorNumber(billData) {
+function getSponsors(billData) { // TODO: change to getSponsors
+  const {
+    activeVersion,
+    amendments: { items },
+    sponsor,
+
+  } = billData;
+  return (
+    1 +
+    items[activeVersion].coSponsors.size +
+    items[activeVersion].multiSponsors.size
+  );
+}
+
+function getSponsorNumber(billData) { // TODO: change to getSponsors
```

Comment:
```suggestion
function getSponsorNumber(billData) {
```
Should this comment be here?